### PR TITLE
New alternative data context + first sub context (Covid)

### PR DESCRIPTION
**Pull request by jmaslek**
As @DidierRLopes and I talked about offline, having alternative data nowadays is all the rage.

Here I add the context + a COVID menu.  Currently this will just show highest/lowest slopes and deaths/cases/death rate for the selected country.

The data is obtained from https://github.com/CSSEGISandData/COVID-19. which is the Johns Hopkins repo.

### CODE REVIEW

**Comment by DidierRLopes** [comment]
crashes badly:
Screenshot 2022-01-06 at 01 56 28

Please, not this date format 🤣 
Screenshot 2022-01-06 at 01 58 14

Not even gonna use the argument that I hate it, but we have in our codebase YYYY-MM-DD pretty much everywhere. So let's stick with that for now, then we can work on a feature flag for the date format preferred by the user.

In any case, due to the type of data we have, we can probably even display like "Jun 2021, Oct 2021, ..". No?

Can we change ylabel to Millions/Thousands/... accordingly? Based on what makes sense for country in question.
Screenshot 2022-01-06 at 02 01 33

File: gamestonk_terminal/alternative/alt_controller.py [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+        if session and gtff.USE_PROMPT_TOOLKIT:
+            choices: dict = {c: {} for c in self.CHOICES}
+
+            self.completer = NestedCompleter.from_nested_dict(choices)
+
+    @staticmethod
+    def print_help():
+        """Print help"""
+        help_str = """
+Alternative Data:
```
Comment (DidierRLopes) [inline gamestonk_terminal/alternative/alt_controller.py:63]: No need for this line, at least we removed it on the last UX iteration

File: gamestonk_terminal/alternative/alt_controller.py [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+>   covid
```
Comment (DidierRLopes) [inline gamestonk_terminal/alternative/alt_controller.py:64]: add some examples of what the user can expect in front

File: gamestonk_terminal/alternative/covid/covid_controller.py [hunk 3]
```diff
@@ -0,0 +1,415 @@
...
+
+    def call_exit(self, _):
+        """Process exit terminal command"""
+        self.queue.insert(0, "quit")
+        self.queue.insert(0, "quit")
+        self.queue.insert(0, "quit")
+
+    def call_reset(self, _):
+        """Process reset command"""
+        self.queue.insert(0, "covid")
```
Comment (DidierRLopes) [inline gamestonk_terminal/alternative/covid/covid_controller.py:164]: the reset can also load the country here

File: gamestonk_terminal/alternative/covid/covid_controller.py [hunk 4]
```diff
@@ -0,0 +1,415 @@
...
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            prog="rates",
+            description="Show historical rates country.",
+        )
+        ns_parser = parse_known_args_and_warn(
+            parser,
+            other_args,
+            export_allowed=EXPORT_BOTH_RAW_DATA_AND_FIGURES,
+            raw=True,
```
Comment (DidierRLopes) [inline gamestonk_terminal/alternative/covid/covid_controller.py:234]: I think this was a significant change to be in a different PR. Because it's not related at all with alternative data. Plus, in that PR we could go back and tackle all functions that needed such.

**Review by DidierRLopes — CHANGES_REQUESTED** [review]
Can we also add a `.gst` file for this?

File: gamestonk_terminal/alternative/covid/covid_controller.py (hunk 4 above)
Comment (jmaslek) [reply gamestonk_terminal/alternative/covid/covid_controller.py:234]: This isn't really a significant change.  Since I added default arguments to the function, it won't break anything and we can go back and revisit where we see as needed.

**Comment by jmaslek** [comment]
> crashes badly: Screenshot 2022-01-06 at 01 56 28

oops

> Not even gonna use the argument that I hate it, but we have in our codebase YYYY-MM-DD pretty much everywhere. So let's stick with that for now, then we can work on a feature flag for the date format preferred by the user.

Fair enough.

> In any case, due to the type of data we have, we can probably even display like "Jun 2021, Oct 2021, ..". No?

I don't think the MPL date formatter is that smart lol

> 
> Can we change ylabel to Millions/Thousands/... accordingly? Based on what makes sense for country in question. Screenshot 2022-01-06 at 02 01 33

Largest number in the entire df is basically 1Mil, so I think leaving this in thousands looks decent enough.

GST file added as well.

**Review by DidierRLopes — APPROVED** [review]
